sky/engine/core/rendering/line/BreakingContextInlineHeaders.h

```diff
@@ -183,6 +185,18 @@ inline bool requiresLineBox(const InlineIterator& it, const LineInfo& lineInfo =
     return notJustWhitespace || isEmptyInline(it.object());
 }
 
+// FIXME: The entire concept of the skipTrailingWhitespace function is flawed, since we really need to be building
+// line boxes even for containers that may ultimately collapse away. Otherwise we'll never get positioned
+// elements quite right. In other words, we need to build this function's work into the normal line
+// object iteration process.
+// NB. this function will insert any floating elements that would otherwise
+// be skipped but it will not position them.
```

Comment:
Yeah, I didn't write this comment.  It was attached to the function when we branched from Blink.  I can remove it if you'd like.